components/src/core/Drawer/DrawerBody.tsx

```diff
@@ -12,64 +14,67 @@ const useStyles = makeStyles({
 });
 
 export type DrawerBodyProps = {
-    activeBackgroundColor?: string;
-    activeFontColor?: string;
-    activeIconColor?: string;
     backgroundColor?: string;
-    chevron?: boolean;
-    fontColor?: string;
-    iconColor?: string;
-    onSelect?: Function;
-    open?: boolean;
-    titleColor?: string;
-};
+} & PXBlueDrawerInheritableGroupProperties;
 
-export const DrawerBody: React.FC<DrawerBodyProps> = (props) => {
-    const classes = useStyles(props);
-    const { backgroundColor } = props;
-    const children = React.Children.toArray(props.children);
+export const DrawerBody: React.FC<DrawerBodyProps> = (bodyProps) => {
+    const classes = useStyles(bodyProps);
+    const { backgroundColor } = bodyProps;
+    const children = React.Children.toArray(bodyProps.children);
     return (
         <div className={classes.root} style={{ backgroundColor }}>
-            {children.map((child: any) => {
+            {children.map((child: any, index) => {
                 if (!child) {
                     return null;
                 }
 
-                const isNavGroup = child.type && child.type.displayName === 'DrawerNavGroup';
-                const groupProps = child.props;
-                return React.cloneElement(
-                    child,
-                    isNavGroup
-                        ? {
-                              activeBackgroundColor: groupProps.activeBackgroundColor || props.activeBackgroundColor,
-                              activeFontColor: groupProps.activeFontColor || props.activeFontColor,
-                              activeIconColor: groupProps.activeIconColor || props.activeIconColor,
-                              backgroundColor: groupProps.backgroundColor || props.backgroundColor,
-                              chevron: groupProps.chevron === undefined ? props.chevron : groupProps.chevron,
-                              fontColor: groupProps.fontColor || props.fontColor,
-                              iconColor: groupProps.iconColor || props.iconColor,
-                              onSelect: props.onSelect,
-                              open: props.open,
-                              titleColor: groupProps.titleColor || props.titleColor,
-                          }
-                        : {}
+                if (child.type && child.type.displayName !== 'DrawerNavGroup') return child;
+                const groupProps: DrawerNavGroupProps = child.props;
+
+                // for any DrawerNavGroup, if a prop is not set (undefined), inherit from the DrawerBody
+                // open and onItemSelect is always determined by DrawerBody
+                return (
+                    <DrawerNavGroup
+                        {...groupProps}
+                        key={index.toString()}
```

Comment:
index alone is not going to be a unique key across the application...do something a little more descriptive